### ISSUE TITLE
Correct the form input border and ring colors.

### ISSUE THREAD

**Issue by tristandunn** [tristandunn/miroha#132]
* New character name input is missing the custom focus ring color.
* Error inputs are missing a focus ring color.
* Unfocused inputs are missing a border color.
